projects/core/src/product/services/search-page-meta.resolver.ts

```diff
@@ -32,10 +38,26 @@ export class SearchPageMetaResolver
     .getRouterState()
     .pipe(map((state) => state.state.params['query']));
 
+  /**
+   * @deprecated since 3.1 we'll use the `BasePageMetaResolver` going forward
+   */
+  // TODO(#10467): Remove deprecated constructors
+  constructor(
+    routingService: RoutingService,
+    productSearchService: ProductSearchService,
+    translation: TranslationService
+  );
+  constructor(
+    routingService: RoutingService,
+    productSearchService: ProductSearchService,
+    translation: TranslationService,
+    basePageMetaResolver?: BasePageMetaResolver
+  );
```

Comment:
Tslint complains in the pipeline. We need to add before constructor:
```
  // tslint:disable-next-line: unified-signatures
```
(here and in other classes)